.github/send_alerts_to_slack.py

```diff
@@ -0,0 +1,95 @@
+"""
+
+Summarize the Frictionless Data JSON error output and if there are any errors, 
+send a summary message to a given Slack channel. Send a more detailed 
+error summary to std out and create a detailed txt file that can be accessed via GitHub
+artifacts.
+
+To run elsewhere, define SLACK_WEBHOOK in your repo's secrets and change the 
+top-level constants defined below. 
+
+`validate-csv.yml` can also be copied verbatim, assuming that the schema is 
+named `datapackage.json`.
+
+Links to...
+- Generate a Slack token
+    https://slack.com/apps/A0F7XDUAZ-incoming-webhooks
+- API for POST requests / formatting text:
+    https://api.slack.com/messaging
+
+
+"""
+
+import json
+import requests
+import os
+
+ERROR_FILE = 'error_summary.json'
+REPO = 'Princeton-CDH/pemm-data'
+CHANNEL = '#slack-webhook-test'
+ERROR_MAX = 5
+
+def main():
+    with open(ERROR_FILE) as f: 
+        j = json.load(f)
+
+    # Create and print message to std out.
+    stdout_string = f"Error Count: { j['error-count'] }\n"
+    for table in j['tables']:
+        if table['error-count'] != 0:
+            stdout_string += f"\tTable: { table['resource-name'] }\n"
+            stdout_string += f"\tError Count: { table['error-count'] }\n"
+            for error in table['errors'][:ERROR_MAX]:
+                stdout_string += f'\t\t{ error["message"] }\n'
+            stdout_string += "\n"
+
+    print(stdout_string)
+
+    # Create a txt file that users can download as an artifact.
+    with open('error_summary.txt', 'w') as f:
+        f.write(stdout_string)
+
+    # If there were no errors, end function and do not send a Slack message.
+    if j['error-count'] == 0:
+        return
+
+    # Create a slack payload.
+    slack_message_prelude = (
+        f"🚨 Goodtables found {j['error-count']} error(s) in the latest sync." +
```

Comment:
I suggest we simplify this — I was looking to see if there were conventions from github or travis-ci we could borrow. Seems like those notices use the issue/pr name / build number as the link. Maybe something like `<goodtables validation failed with X errors>` where that text is the link, and no additional text is needed.